src/rf/apps/home/tests.py

```diff
@@ -92,6 +92,17 @@ def test_create_layer_no_permission(self):
         response = self.save_layer(layer, user)
         self.assertEqual(response.status_code, 401)
 
+    def test_create_layer_date_errors(self):
+        # start is after end
+        layer = {
+            'name': 'n',
+            'capture_start': '2015-08-15',
+            'capture_end': '2015-08-14',
+        }
+        user = self.user_models[self.logged_in_user]
+        response = self.save_layer(layer, user)
+        self.assertEqual(response.status_code, 403)
```

Comment:
True. BTW, I'm not validating that description, organization, etc. are non-empty on the backend because I'm thinking about removing those client-side checks.